features/step_definitions/files_steps.rb

```diff
@@ -26,7 +41,6 @@
 end
 
 
-Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |file_name, content|
-  expect(uncommitted_files).to eql [file_name]
-  expect(IO.read file_name).to eql content
+Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |name, content|
```

Comment:
Haha well StackOverflow is hardcore then! I think it's a nice approach to remember because it works anywhere. Like:
```bash
find features/ -type f -exec sed -i 's/find/replace/g' '{}' \;
```

I see, that's a good way of doing it and a nice plugin, thanks! And it's definitely fast.